docfx_project/docfx.json

```diff
@@ -59,7 +61,8 @@
     "globalMetadataFiles": [],
     "fileMetadataFiles": [],
     "template": [
-      "default"
+      "default",
+	  "./template/playwright-sharp"
```

Comment:
Do we need this?

Reply:
No, I have removed the refrence to "./template/playwright-sharp"